pkg/profile/profile.go

```diff
@@ -92,12 +112,20 @@ func (ps *ProfileService) GetUsersGameInfo(ctx context.Context, r *pb.UsersGameI
 	var infos []*pb.UserGameInfo
 
 	for _, uuid := range r.Uuids {
-		user, err := ps.userStore.GetByUUID(ctx, uuid)
-		if err == nil {
+		user, userStoreErr := ps.userStore.GetByUUID(ctx, uuid)
+		avatarUrl := user.AvatarUrl()
```

Comment:
remember to check `userStoreErr`, otherwise `user` may be `nil` and this may panic

Reply:
Good catch, I believe I have fixed it.